config/locales/en.yml

```diff
@@ -1133,7 +1133,7 @@ en:
       this_is_private_community: "You need to sign up before you can view the content."
     invitation_form:
       email: "Email address(es)"
-      email_with_limit: "Email addresses (max. %{invitation_limit})"
+      email_with_limit: "Email addresses (max. %{invitation_limit} per day)"
```

Comment:
After discussing with Thomas & Antti, we decided to remove all explicit info about daily limits since only few admins can see it (and they see also this linked KB article)